staging/src/k8s.io/client-go/tools/clientcmd/client_config_test.go

```diff
@@ -624,6 +624,26 @@ func TestCreateMissingContext(t *testing.T) {
 	}
 }
 
+func TestCreateAuthConfigExecInstallHintCleanup(t *testing.T) {
+	config := createValidTestConfig()
+	clientBuilder := NewNonInteractiveClientConfig(*config, "clean", &ConfigOverrides{
+		AuthInfo: clientcmdapi.AuthInfo{
+			Exec: &clientcmdapi.ExecConfig{
+				APIVersion:  "client.authentication.k8s.io/v1alpha1",
+				Command:     "some-command",
+				InstallHint: "some install hint with \x1b[1mcontrol chars\x1b[0m\nand a newline",
```

Comment:
I think a more exhaustive test would be good, maybe a unit test of just that function.

Reply:
Thanks for the quick review @lavalamp. I tried to add a unit test for this function here: 139e7ee16399a82a4ac21633c64d4a86321d22fe.